scripts/prepare_ml_data.py

```diff
@@ -48,6 +48,7 @@
 @click.option(
     "--overwrite_batches",
     default=False,
+    is_flag=True,
```

Comment:
just a thought - should we sel this to false - so we dont accidently delete a load of batches

Reply:
Don't worry: the default is False!  (i.e. the default behaviour is to _not_ overwrite batches).

The `is_flag=True` flag is just telling `click` that this is a flag (i.e. a boolean).